src/code/gfxprint.c

```diff
@@ -285,58 +283,61 @@ void GfxPrint_PrintChar(GfxPrint* this, u8 c) {
     }
 }
 
-void GfxPrint_PrintStringWithSize(GfxPrint* this, const void* buffer, size_t charSize, size_t charCount) {
+void GfxPrint_PrintStringWithSize(GfxPrint* this, const void* buffer, u32 charSize, u32 charCount) {
     const char* str = (const char*)buffer;
-    size_t count = charSize * charCount;
+    u32 count = charSize * charCount;
```

Comment:
Essentially all of the size_t arguments here had to be changed to make it compatible with Print_Callback which uses u32 instead of size_t, so I went with u32 for the variable as well for consistency.
So I think it's out of scope for this PR but PrintCallback itself could be changed eventually if we want to use size_t